src/elm/arithmetic.ts

```diff
@@ -381,7 +382,7 @@ export class Power extends Expression {
 }
 
 export class MinValue extends Expression {
-  static readonly MIN_VALUES = {
+  static MIN_VALUES = {
```

Comment:
Is there a reason to take out the `readonly` qualifier here? The value of `MIN_VALUES` never gets re-assigned. I saw that `MAX_VALUES` below still has `readonly`, so figured I'd ask about the change

Reply:
Nope, just added readonly back!

Reply:
@csenn - I'm taking a look at this PR again, as it would be great to get it resolved. I noticed that you said you added the `readonly` back on line 385, but when I look at the current code on this branch, I don't see it. Perhaps you have not pushed it back up?